.github/workflows/check-pr.yaml

```
@@ -11,7 +11,7 @@ jobs:
     name: Check commit message
     runs-on: ubuntu-22.04
     # Skip on draft, check on draft -> ready
-    if: github.repository_owner == 'Samsung' && github.event.pull_request.draft == false
+    if: github.event.pull_request.draft == false
```

```suggestion
    if: github.repository_owner == 'Samsung' && github.event.pull_request.draft == false
```

Let's revert this change.

---

I asked @dayo09  to revert this change, because the condition would be for not triggering this workflow in the forked repo.